hypershift-operator/main.go

```diff
@@ -144,7 +144,7 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 		Scheme:                        hyperapi.Scheme,
 		MetricsBindAddress:            opts.MetricsAddr,
 		Port:                          9443,
-		LeaderElection:                true,
+		LeaderElection:                false,
```

Comment:
Looks like maybe an unintentional debug commit? I'd love to have a flag for this btw